sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -24,6 +24,8 @@ import javax.xml.bind.DatatypeConverter
 
 import scala.annotation.tailrec
 
+import sun.util.calendar.CalendarSystem
```

Comment:
Seems reasonable - will also allow us to hide the julian calendar usage for old dates.